bayesian/default_config.py

```diff
@@ -24,6 +24,13 @@
 # length of token life in seconds
 API_TOKEN_LIFETIME = 3600
 
+# license synonyms
+# /<url>/<org>/<repo>/<branch>/<dir/file>
+LIC_SYNONYMS_URL = '/'.join(['https://raw.githubusercontent.com', 'fabric8-analytics',
```

Comment:
I realize that this is very convenient, but it also can make debugging harder as nobody will be sure what version exactly was used by the service when a problem occurred. Just saying :)

Reply:
yes, I agree with you but this is just a JSON file contains synonyms list if it has some problem it will occur in `fabric8-analytics-license-analysis` first.